pkg/asset/installconfig/aws/session.go

```diff
@@ -21,17 +21,29 @@ func GetSession() (*session.Session, error) {
 	ssn := session.Must(session.NewSessionWithOptions(session.Options{
 		SharedConfigState: session.SharedConfigEnable,
 	}))
+
+	sharedCredentialsProvider := &credentials.SharedCredentialsProvider{}
 	ssn.Config.Credentials = credentials.NewChainCredentials([]credentials.Provider{
 		&credentials.EnvProvider{},
-		&credentials.SharedCredentialsProvider{},
+		sharedCredentialsProvider,
 	})
-	_, err := ssn.Config.Credentials.Get()
+
+	creds, err := ssn.Config.Credentials.Get()
+	if err == nil {
+		switch creds.ProviderName {
+		case "SharedCredentialsProvider":
+			logrus.Infof("Credentials loaded from the %q profile in file %s", sharedCredentialsProvider.Profile, sharedCredentialsProvider.Filename)
```

Comment:
nit: i think it's also appropriate to quote the file name because file names can have spaces ??